bower.json

```diff
@@ -1,26 +1,19 @@
 {
-  "name": "purescript-arraybuffer-types",
-  "homepage": "https://github.com/purescript-contrib/purescript-arraybuffer-types",
-  "description": "PureScript type definitions for JavaScript ArrayBuffers",
-  "keywords": [
-    "purescript"
-  ],
-  "license": "MIT",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/purescript-contrib/purescript-arraybuffer-types.git"
-  },
-  "ignore": [
-    "**/.*",
-    "bower_components",
-    "node_modules",
-    "output",
-    "bower.json",
-    "gulpfile.js",
-    "package.json"
-  ],
-  "devDependencies": {
-    "purescript-console": "^5.0.0",
-    "purescript-effect": "^3.0.0"
-  }
+    "name": "purescript-arraybuffer-types",
+    "license": [
+        "MIT"
+    ],
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/purescript-contrib/purescript-arraybuffer-types"
+    },
+    "ignore": [
+        "**/.*",
+        "node_modules",
+        "bower_components",
+        "output"
+    ],
+    "dependencies": {
+        "purescript-psci-support": "^v5.0.0"
```

Comment:
This should be in `devDependencies`; if you want to generate it from the `spago.dhall` file, then we can add a separate `dev.dhall` file that includes `psci-support` and remove all non-dev dependencies from the main `spago.dhall`.

Reply:
Ok I'll do it that way in the future, thanks for the advice.